src/hash_table.py

```diff
@@ -0,0 +1,38 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+class HashTable(object):
+    """Implement a hash table class in Python."""
+    def __init__(self, iterable=None):
```

Comment:
Your HashTable's init should take a "size" argument, which determines the number of buckets. Currently, you're arbitrarily setting this to 10.